pkgs/by-name/rc/rclone-ui/package.nix

```diff
@@ -0,0 +1,87 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+
+  cargo-tauri,
+  nodejs,
+  npmHooks,
+  fetchNpmDeps,
+
+  pkg-config,
+  wrapGAppsHook3,
+
+  openssl,
+  libsoup_2_4,
+  webkitgtk_4_1,
+  glib-networking,
+  libappindicator,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "rclone-ui";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "rclone-ui";
+    repo = "rclone-ui";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-KTi/vCHiZVRAmQAiVXSWHCTTv1NnsvM5UZg8cpuFbRQ=";
+  };
+
+  npmDeps = fetchNpmDeps {
+    name = "${finalAttrs.pname}-${finalAttrs.version}-npm-deps";
+    inherit (finalAttrs) src;
+    forceGitDeps = true;
+    hash = "sha256-18QkqqYS1kGY701FbFBHLvr5WBkJzxFgR9VMnydeelY=";
+  };
+
+  cargoRoot = "src-tauri";
+  buildAndTestSubdir = finalAttrs.cargoRoot;
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-o21of2eS2KZtg1U1E6RwdaA8jGhEVzg7HkgOv1k5wxI=";
+
+  # Disable tauri bundle updater, can be removed when #389107 is merged
+  patches = [ ./remove_updater.patch ];
+
+  postPatch = ''
+    substituteInPlace src-tauri/Cargo.toml \
+       --replace-fail 'name = "app"' 'name = "${finalAttrs.pname}"'
+  '';
+
+  nativeBuildInputs = [
+    npmHooks.npmConfigHook
+    nodejs
+
+    cargo-tauri.hook
+
+    pkg-config
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    openssl
+    libsoup_2_4
+    webkitgtk_4_1
+    glib-networking
+    libappindicator
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/${finalAttrs.pname} \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libappindicator ]}
```

Comment:
For this part, I'm unsure, I might not be the best person for reviewing this part.

Reply:
Can we use patchelf instead? That doesn't polute LD_LIBRARY_PATH for execed binaries.